core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -309,6 +311,8 @@ oppia.directive('conversationSkin', [
           UserService.getUserInfoAsync().then(function(userInfo) {
             $scope.isLoggedIn = userInfo.isLoggedIn();
           });
+          $templateCache.put(
```

Comment:
Maybe add a comment here explaining why we put stuff into `$templateCache`, it can be short, just simple explanation.

Reply:
Done.